src/lib.rs

```diff
@@ -416,10 +435,19 @@ impl Database {
     {
         let (src, face_index) = self.face_source(id)?;
         match &*src {
-            #[cfg(feature = "fs")]
+            #[cfg(all(feature = "fs", not(feature="memmap2")))]
+            Source::File(ref path) => {
+                let mut file = std::fs::File::open(path).ok()?;
+                let mut buffer = Vec::new();
+                std::io::Read::read_to_end(&mut file, &mut buffer).ok()?;
```

Comment:
You can use [`std::fs::read()`](https://doc.rust-lang.org/stable/std/fs/fn.read.html) here.